.github/workflows/ci.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
```

Comment:
Probably none of these are needed

Reply:
I'll remove them... they will be blank anyway...